pulpcore/cli/core/repository.py

```diff
@@ -32,3 +47,49 @@ def repository(ctx: click.Context, pulp_ctx: PulpCLIContext) -> None:
 
 repository.add_command(list_command(decorators=filter_options))
 repository.add_command(version_command(decorators=[], list_only=True))
+
+
+@repository.command()
+@pass_repository_context
+@pass_pulp_context
+@repositories_option
+@pulp_option(
+    "--all",
```

Comment:
Is this a good name, or too ambiguous?

@gerrod3 @ggainey

Reply:
I think it is fine, the command would be `pulp repository reclaim-space --all`. An alternative would be `all-repos`, but it should be obvious what this field does looking at the help message.